src/dbnode/storage/index.go

```diff
@@ -986,6 +1012,32 @@ func (i *nsIndex) query(
 	results index.BaseResults,
 	opts index.QueryOptions,
 	execBlockFn execBlockQueryFn,
+) (bool, error) {
+	ctx, sp := ctx.StartTraceSpan(tracepoint.NSIdxSubQuery)
+	sp.LogFields(
```

Comment:
Dunno convention but IMO child spans don’t need to log fields that are the same as the parent. 

ie all the fields in this log are already logged by Aggregate/Query

Reply:
Yeah so I'm kind of unsure of this too. On the one hand you're just duplicating data. On the other hand, it's easier to grok with the info right there in the span. Let me read up on the convention.

Reply:
Yeah that’s fair. I’m happy to leave in if it helps readability.

Reply:
So there's not really any best practices around this (at least not that I could find), but I think it's better to leave these in here for readability.